exporter/importables.go

```diff
@@ -582,6 +612,15 @@ var resourcesMap map[string]importable = map[string]importable{
 						ID:       eitherString(value, defaultValue),
 					})
 				}
+				if typ == "fixed" && (strings.HasPrefix(k, "spark_conf.") || strings.HasPrefix(k, "spark_env_vars.")) {
```

Comment:
For my understanding can you explain this a bit more, why `fixed` is required?

Reply:
Because there are other types, like, `regexp` - in this case we won't be able to handle it